pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -1042,7 +1041,6 @@ public void maxpeersOptionMustBeUsed() {
   }
 
   @Test
-  @Ignore
```

Comment:
Reminder to get the other two comments posted here: https://github.com/PegaSysEng/pantheon/pull/1218#discussion_r272271686

Reply:
It is ok.